spec/tasks/gen/task_spec.cr

```diff
@@ -0,0 +1,49 @@
+require "../../spec_helper"
+
+include CleanupHelper
+include GeneratorHelper
+
+private def run_with_args(args : Array(String))
+  generator = Gen::Task.new
+  generator.output = IO::Memory.new
+  generator.print_help_or_call args
+  generator.output
+end
```

Comment:
This method feels like it _could_ be helpful in other tests, but I don't want to make that kind of change without a discussion. For a more generic usage:

```crystal
def run_with_args(task : LuckyCli::Task.class, args : Array(String))
  runner = task.new
  runner.output = IO::Memory.new
  runner.print_help_or_call args
  runner.output # maybe with .to_s?
end
```